src/clp_notification_monitor/compression_buffer/compression_buffer.py

```diff
@@ -1,8 +1,20 @@
 from datetime import datetime, timedelta
 from logging import Logger
-from pathlib import PurePath
+from pathlib import Path, PurePath
 from threading import Condition, Lock
-from typing import List, Optional
+from typing import Dict, List, Optional
+
+
+class SingleBuffer:
+    def __init__(self) -> None:
+        self.path_list: List[PurePath] = []
+        self.total_buffer_size: int = 0
+        self.first_path_timestamp: Optional[datetime] = None
+
+    def clear(self) -> None:
+        self.path_list = []
```

Comment:
```suggestion
        self.path_list.clear()
```
Nit: better locality